graders/r/plr/DESCRIPTION

```diff
@@ -0,0 +1,18 @@
+Package: plr
+Title: Utility Functions for 'PrairieLearn' and R
+Version: 0.1.0
+Date: 2021-08-17
```

Comment:
We could update the version, but it no longer matters much as it is in the same repo.  A Date: is optional for R so we could remove it.  (I tend to keep them, just as a reminder to myself...)

Reply:
Updated to date of version bump